renderer/components/Home/WalletSettingsFormLocal.js

```diff
@@ -48,6 +61,32 @@ class WalletSettingsFormLocal extends React.Component {
 
           <DataRow left={<FormattedMessage {...messages.chain} />} py={2} right={chain} />
           <DataRow left={<FormattedMessage {...messages.network} />} py={2} right={network} />
+
+          <Bar variant="light" />
+          <DataRow
```

Comment:
Lets move this whole section to just above or below the autopilot settings and give the section a heading like `Connections` or `Advanced`

Reply:
done